example/discord.js

```diff
@@ -0,0 +1,25 @@
+/* install ytdl-core before running this! */
+/* npm install --save ytdl-core */
```

Comment:
This note should be about `discord.js`
`ytdl=core` is assumed to be already installed, and referenced with `require('..')` since this example is inside the repo for it.